UIKitTemplate/UIKitTemplate/Views/Feed/PostHeaderView.swift

```diff
@@ -0,0 +1,79 @@
+// PostHeaderView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Верхняя панель поста
+final class PostHeaderView: UIView {
+    // MARK: - Constants
+
+    private enum Constants {
+        static let menuButtonImage = "ellipsis"
+    }
+
+    // MARK: - Visual Components
+
+    private let avatarImageView = AvatarImageView()
+
+    private let accountNameLabel: UILabel = {
+        let label = UILabel()
+        label.font = .verdanaBold(ofSize: 12)
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+
+    private let menuButton: UIButton = {
+        let button = UIButton()
+        button.setImage(
+            UIImage(systemName: Constants.menuButtonImage)?.withRenderingMode(.alwaysTemplate),
```

Comment:
```suggestion
            UIImage(systemName: Constants.menuButtonImage)
```
Он же по умолчанию стоит